### PR TITLE
[onkyo] Support for more audio streams through the HTTP audio servlet

### PR DESCRIPTION
**Pull request by lolodomo**
Related to #15113

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@dalgwen for information and feedback

**Comment by dalgwen** [comment]
> @dalgwen for information and feedback

LGTM : the new capabilities are used as intended.

**Comment by lolodomo** [comment]
Do we set the timeout for 5 or to 10 for more security ?

**Comment by lolodomo** [comment]
By security, I finally set the timeout to 10s.

@kaikreuzer : this is ready for a final review.

File: bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoHandlerFactory.java [hunk 1]
```diff
@@ -77,14 +78,12 @@ protected ThingHandler createHandler(Thing thing) {
...
-            OnkyoHandler handler = new OnkyoHandler(thing, upnpIOService, audioHTTPServer, callbackUrl,
-                    stateDescriptionProvider);
-            if (callbackUrl != null) {
-                @SuppressWarnings("unchecked")
-                ServiceRegistration<AudioSink> reg = (ServiceRegistration<AudioSink>) bundleContext
-                        .registerService(AudioSink.class.getName(), handler, new Hashtable<>());
-                audioSinkRegistrations.put(thing.getUID().toString(), reg);
-            }
+            OnkyoHandler handler = new OnkyoHandler(thing, upnpIOService, stateDescriptionProvider);
+            OnkyoAudioSink audiSink = new OnkyoAudioSink(handler, audioHTTPServer, callbackUrl);
```
Comment (jlaur) [inline bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoHandlerFactory.java:82]: Variable name: audioSink

**Review by jlaur — CHANGES_REQUESTED** [review]
Minor typo found.

File: bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoHandlerFactory.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoHandlerFactory.java]: Fixed

**Review by kaikreuzer — APPROVED** [review]
lgtm, thanks!

**Pull request merged**